.travis.yml

```diff
@@ -11,9 +11,9 @@ python:
 cache: pip
 
 install:
+  - bash .travis/install_terraform.sh
```

Comment:
This was moved up to avoid wasting cycles if this relatively simple script fails.